srcpkgs/python3-ocrmypdf/template

```diff
@@ -0,0 +1,38 @@
+# Template file for 'python3-ocrmypdf'
+pkgname=python3-ocrmypdf
+version=16.10.0
+revision=1
+build_style=python3-pep517
+hostmakedepends="python3-setuptools python3-wheel python3-poetry-core"
+makedepends="hatch-vcs"
+depends="python3-deprecation img2pdf python3-packaging python3-pdfminer.six
+ python3-pikepdf python3-Pillow python3-pluggy python3-reportlab python3-rich
+ tesseract-ocr ghostscript unpaper pngquant jbig2enc"
+short_desc="Add OCR text layer to scanned PDF files"
+maintainer="Mateusz Sylwestrzak <slymattz@gmail.com>"
+license="MPL-2.0"
+homepage="https://github.com/ocrmypdf/OCRmyPDF"
+distfiles="${PYPI_SITE}/o/ocrmypdf/ocrmypdf-${version}.tar.gz"
+checksum=d5b907a7b92951f1f3617f0f5ca002d866143d94fd168546a70e51756bf6412e
+
+post_patch() {
+	# point to tesseract-ocr binary found in PATH
+	vsed -i src/ocrmypdf/_exec/tesseract.py \
+		-e "s|return TesseractVersion(get_version('tesseract'|return TesseractVersion(get_version('tesseract-ocr'|g" \
+		-e "s|\['tesseract'|\['tesseract-ocr'|g"
+}
+
+pre_configure() {
+	# this is an optional dep
+	vsed -i pyproject.toml \
+		-e '/^\s*"pi-heif"/d'
+
+	# ERROR Backend 'hatchling.build' is not available.
+	# let's then remove this automatic versioning tool and rely on a manual entry
```

Comment:
This is probably a consequence of putting the hatch build dependency in the wrong place. It belongs in `hostmakedepends`.